blob/helper.go

```diff
@@ -1,15 +1,26 @@
 package blob
 
 import (
+	"bytes"
+	"sort"
+
+	"github.com/tendermint/tendermint/types"
+
 	"github.com/celestiaorg/celestia-app/pkg/appconsts"
 	"github.com/celestiaorg/celestia-app/pkg/shares"
+
+	"github.com/celestiaorg/celestia-node/share"
 )
 
-// sharesToBlobs takes raw shares and converts them to blobs.
-func sharesToBlobs(rawShares [][]byte) ([]*Blob, error) {
+// SharesToBlobs takes raw shares and converts them to the blobs.
+func SharesToBlobs(rawShares []share.Share) ([]*Blob, error) {
 	if len(rawShares) == 0 {
 		return nil, ErrBlobNotFound
 	}
+	rawShares, err := removePadding(rawShares)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Is there a reason why parse shares does not removed padding automatically? It feels kinda ugly that users have to do it themselves + it's unnecessary allocations. 

I know it's more of a question to the app team, but maybe you know.